pykg2tbl/query.py

```diff
@@ -0,0 +1,185 @@
+import logging
+from abc import ABC, abstractmethod
+from typing import List
+
+import pandas as pd
+
+from pykg2tbl.exceptions import (
+    NoCompatibilityChecker,
+    NotASubClass,
+    WrongInputFormat,
+)
+
+log = logging.getLogger(__name__)
+
+
+class QueryResult(ABC):
```

Comment:
why not subclass from `SPARQLWrapper.QueryResult`?

Reply:
@marc-portier ?